app/views/idv/how_to_verify/show.html.erb

```diff
@@ -0,0 +1,79 @@
+<% title t('doc_auth.headings.how_to_verify') %>
+
+<%= render PageHeadingComponent.new.with_content(t('doc_auth.headings.how_to_verify')) %>
+<p><%= t('doc_auth.info.how_to_verify') %></p>
+
+<%= simple_form_for(
+      @idv_how_to_verify_form,
+      html: { autocomplete: 'off' },
+      method: :put,
+      url: idv_how_to_verify_url,
+    ) do |f|
+%>
+  <div role="group" aria-label="<%= t('doc_auth.headings.how_to_verify') %>">
+    <%= radio_button_tag(
+          'idv_how_to_verify_form[selection]',
+          Idv::HowToVerifyController::REMOTE,
+          false,
+          class: 'usa-radio__input usa-radio__input--tile',
+        ) %>
+    <%= label_tag(
+          'idv_how_to_verify_form_selection_remote',
+          class: 'usa-radio__label usa-radio__label--illustrated usa-radio__label-illustrated-large',
+        ) do %>
+          <%= image_tag asset_url('idv/remote.svg'),
+                        width: 88,
+                        height: 92,
+                        class: 'usa-radio__image',
+                        alt: t('image_description.laptop_and_phone')
+          %>
+          <div class="usa-radio__label--text">
+            <span class="usa-tag text-light bg-primary-darker margin-bottom-2">
+              <%= t('doc_auth.tips.most_common') %>
+            </span>
+            <p class="text-gray-70"><%= t('doc_auth.headings.verify_online') %></p>
```

Comment:
At the very least I'd suggest using the semantic token name `text-ink` rather than the specific color grade, though I'd also question why we're deviating from the standard heading color for the component.

Also, I think the default blue only applies for the heading and not the description text. We might be missing the description wrapper element which should style it as the default (grayscale) color.

https://federalist-340d8801-aa16-4df5-ad22-b1e3e731098e.sites.pages.cloud.gov/site/18f/identity-design-system/form-fields/#tile

Reply:
Thanks @allthesignals! One of the reasons for using the heading inside the radio button in dark grey is because the icons in the card are also grey. For the purposes of this experiment, we can make minimal changes to the existing component and keep the blue headings. And I can bring up in a future design system discussion potential variants that we could add to the design system @aduth